src/dash-table/syntax-tree/lexeme/expression.ts

```diff
@@ -6,6 +6,7 @@ import { ISyntaxTree } from 'core/syntax-tree/syntaxer';
 const FIELD_REGEX = /^{(([^{}\\]|\\.)+)}/;
 const STRING_REGEX = /^(('([^'\\]|\\.)*')|("([^"\\]|\\.)*")|(`([^`\\]|\\.)*`))/;
 const VALUE_REGEX = /^(([^\s'"`{}()\\]|\\.)+)(?:[\s)]|$)/;
+const PERMISSIVE_VALUE_REGEX = /^(([^'"`{}()\\]|\\.)+)(?:$)/;
```

Comment:
```suggestion
const PERMISSIVE_VALUE_REGEX = /^(([^'"`{}()\\]|\\.)+)$/;
```
Minor, no point not capturing the `$` right?